src/features/expense/ui/DayExpenseList/DayExpenseList.tsx

```
@@ -0,0 +1,18 @@
+import { useExpenseStore } from '@/features/expense/model/expenseStore';
+
+const DayExpenseList = () => {
+  const month = useExpenseStore((state) => state.month);
```

ESLint

[error] 'month' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)